### PR TITLE
Mateuszwalendzik/ch19193/space app update usage metrics view with

### PR DESCRIPTION
**Pull request by mateuszJS**
image

- update charts view
Note: loading state will be done in another PR
Todo: Update the space-ui lib

### CODE REVIEW

File: public/electron/clients/billing/endpoints/account.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * @property {number} bandwidth
+ */
+
+/**
+ * @this {import('../client.js')}
+ * @param {Object} payload
+ * @param {string} payload.token
+ * @returns {import('axios').AxiosResponse<CurrentUsageInfo>}
+ */
+function getCurrent(payload) {
```
Comment (giankotarola) [inline public/electron/clients/billing/endpoints/account.js:15]: `getCurrentUsageInfo` could be a better naming

File: public/electron/clients/billing/client.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * @param {Object} config - Config for billing client.
+ * @param {string} config.baseURL - Base url.
+ * @param {number} [config.timeout] - Base timeout in ms.
+ * @param {Object} [config.headers] - Base headers.
+ */
+function BillingClient({
+  baseURL,
+  headers,
+  timeout,
+}) {
```
Comment (giankotarola) [inline public/electron/clients/billing/client.js:16]: creating another client could not be necessary. We can create endpoints for billing and set the correct URL to those endpoints. In that way, we don’t repeat a new client class creation

Comment (mateuszJS) [reply public/electron/clients/billing/client.js]: fixed 1b83e2e4658207e0880c743e1d3a51d64249ab35

File: public/electron/clients/billing/endpoints/account.js (hunk 1 above)
Comment (mateuszJS) [reply public/electron/clients/billing/endpoints/account.js]: fixed 1b83e2e4658207e0880c743e1d3a51d64249ab35

**Review by giankotarola — APPROVED** [review]

**Pull request merged**